virtcontainers/qemu.go

```diff
@@ -1225,12 +1226,11 @@ func genericBridges(number uint32, machineType string) []Bridge {
 	var bt bridgeType
 
 	switch machineType {
-
 	case QemuQ35:
 		// currently only pci bridges are supported
 		// qemu-2.10 will introduce pcie bridges
 		fallthrough
-	case QemuPC:
+	case QemuPC, QemuVirt:
```

Comment:
@sboeuf you should be using a -device pcie-pci-bridge along the lines of 

`-device pcie-pci-bridge,id=<id>,bus=pcie.0`

What does this end up being in this case? pci-bridge?

What is the b.Type for virt?

https://github.com/kata-containers/runtime/pull/805/commits/ecb55ec33de135e84bf3a0408e885acf2a70c33e#diff-f02dbf94518fc0fe68a41a78d181b390R1202

Reply:
@mcastelino this is part of discussions that we need to have in order to improve things. But I thought that for now, we should at least get this working the same way it works with `pc`, don't you think?

This ends up doing adding a simple `-device pci-bridge` in this case.

Reply:
@sboeuf as we use the latest QEMU for virt we should be using pcie-pci-bridge for now. And for Q35 based on the QEMU version used that too should use pcie-pci-bridge, but that is for later.